demo-shell-ng2/package.json

```diff
@@ -62,6 +62,7 @@
     "@angular/platform-browser-dynamic": "2.2.2",
     "@angular/router": "3.2.2",
     "@angular/upgrade": "2.2.2",
+    "@angular/material": "^2.0.0-beta.1",
```

Comment:
I suggest pinning to the specific version. Who knows what they change in the upcoming beta.